src/js/actions/tools.js

```diff
@@ -704,6 +709,25 @@ define(function (require, exports) {
         installShapeDefaults, "layers.resetLayers"];
 
     /**
+     * Enter the free transform path mode modal tool state. 
+     * We do not care if this event fails, and it may take a long time to return 
+     * 
+     * @return {Promise}
+     */
+    var enterPathModalState = function () {
+        descriptor.batchPlayObjects([vectorMaskLib.enterFreeTransformPathMode()],
+                { synchronous: false })
+            .catch(function () {
+                // Silence the errors here since we cannot guarantee that PS will not throw an error
+                // when we end the modal tool state in a valid case
+            });
+        return Promise.resolve();
```

Comment:
I'm skeptical that we don't want to return this playObject promise.  I get that we don't want it to bubble out an exception, but seems like it could at least wait for it to resolve.

Also, it seems like we'd at least want to *log* the errors.  It seems like over time our goal is to try our hardest not to make ps calls that will error, so we'll want to use the messages as diagnostics.  eh?